hail/src/main/scala/is/hail/expr/ir/IR.scala

```diff
@@ -430,6 +430,7 @@ final case class CombOp(i1: Int, i2: Int, aggSig: PhysicalAggSig) extends IR
 final case class ResultOp(startIdx: Int, aggSigs: IndexedSeq[PhysicalAggSig]) extends IR
 final case class CombOpValue(i: Int, value: IR, aggSig: PhysicalAggSig) extends IR
 final case class AggStateValue(i: Int, aggSig: AggStateSig) extends IR
+final case class InitFromSerializedValue(i: Int, value: IR, aggSig: PhysicalAggSig) extends IR
```

Comment:
I think this probably only needs the AggStateSig, since we don't need the op to serialize or deserialize.